.github/workflows/checker.yml

```diff
@@ -1,31 +1,46 @@
-name: "Checker"
-on:  # yamllint disable-line rule:truthy
+---
+name: Checker
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_dispatch:
   schedule:
     - cron: "0 4 * * 5"
-  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: false
+
+permissions:
+  contents: read
+
+env:
+  PYTHON_VERSION: "3.13"
 
 jobs:
-  checker:
-    name: Checker
-    runs-on: ubuntu-24.04
+  search:
+    name: Search
+    runs-on: ubuntu-24.04-arm
     steps:
```

Comment:
I think that's a very good idea to use an ARM image for the CI jobs, so we discover any problems early on that we don't have on our developer desktop systems :+1: